Scripts/RemediationScripts/Remediate-EnableAADOnlyAuthenticationForSQLServers.ps1

```diff
@@ -669,20 +762,22 @@ function Disable-ADAdminForSqlServers
                          @{Expression={$_.ServerName};Label="Server Name";Width=20;Alignment="left"},
                          @{Expression={$_.ResourceType};Label="Resource Type";Width=30;Alignment="left"},
                          @{Expression={$_.IsSynapseWorkspace};Label="Is Synapse Workspace?";Width=25;Alignment="left"},
-                         @{Expression={$_.EmailId};Label="Display Name";Width=35;Alignment="left"}
+                         @{Expression={$_.IsAADAdminPreviouslyConfigured};Label="Is AAD Admin Previously Configured?";Width=25;Alignment="left"},
+                         @{Expression={$_.EmailId};Label="Email Id";Width=35;Alignment="left"}
 
     $colsProperty2 = @{Expression={$_.ResourceGroupName};Label="Resource Group Name";Width=25;Alignment="left"},
                          @{Expression={$_.ServerName};Label="Server Name";Width=20;Alignment="left"},
                          @{Expression={$_.ResourceType};Label="Resource Type";Width=30;Alignment="left"},
-                         @{Expression={$_.IsSynapseWorkspace};Label="Is Synapse Workspace?";Width=25;Alignment="left"}
+                         @{Expression={$_.IsSynapseWorkspace};Label="Is Synapse Workspace?";Width=25;Alignment="left"},
```

Comment:
is this synapse information required for end user?

Reply:
It has been here from the previous script, so did not modify it. In case of any errors it might help user.